content/offices/brisbane.mdx

```diff
@@ -39,9 +39,9 @@ aboutUs: >
   presenters](https://tv.ssw.com/tag/damian-brady,adam-stephensen,brendan-richards,igor-goldobin/),
   and we pride ourselves on our teaching expertise with highly effective
   trainers. We've been instrumental in effecting change with everything from
-  [software audits](https://www.ssw.com.au/ssw/Consulting/Software-Audit.aspx)
+  [software audits](https://www.ssw.com.au/consulting/software-audit)
   to [training
-  courses](https://www.ssw.com.au/ssw/Company/Offices/brisbane/ssw/Events/), to
+  courses](https://www.ssw.com.au/offices/brisbane), to
```

Comment:
this looks like a bug - I would expect that it links to `/events` with a filter for location=brisbane

Reply:
done